shell/app/layout/pages/page-container/components/sidebar.scss

```diff
@@ -58,3 +58,10 @@
     }
   }
 }
+
+.message-badge .ant-badge-count {
+  right: 0 !important;
+  background-color: transparent;
```

Comment:
According to PD's opinion, the red wrapper is still needed.

Reply:
ok